components/cap-arrowRight.js

```diff
@@ -0,0 +1,27 @@
+import { LitElement, svg, css } from "../lib/lit.js";
+import { baseStyles } from "../lib/wc-base.js";
+
+export class CapArrowRight extends LitElement {
```

Comment:
I think we could rename this `CapArrow` if we wanted it to be flexible for other uses (eg, an arrow pointing left). I am not sure just yet how many other parts of the CAP site we will be carrying over that will need it, however, so for now it seemed to make the most sense to name it CapArrowRight and to make it a web component that accepts no properties. 

If we do need other orientations, I think we can easily add an "orientation" or "direction" property, and assign right as the default value if not otherwise provided.